emscripten-releases-tags.txt

```diff
@@ -9,17 +9,18 @@
     "latest-releases-upstream": "latest"
   },
   "releases": {
+    "2.0.26-lto": "b92ba43f3ac92ab6f1ce6136a8c5969b68ba6968",
     "2.0.26": "823d37b15d1ab61bc9ac0665ceef6951d3703842",
     "2.0.25": "f6f001b08fbb67935379cf13d17fd9bfdbaff791",
     "2.0.24": "6ab7fc5622a67e6111d07c4ba61c8d3c8fc33ed2",
-    "2.0.23-lto": "3f6dbb899f61fab52e4574beb4f7c8382658aa20",
     "2.0.23": "77b065ace39e6ab21446e13f92897f956c80476a",
+    "2.0.23-lto": "3f6dbb899f61fab52e4574beb4f7c8382658aa20",
     "2.0.22": "6465a9acb820207acf7da44661a7de52d0a1ae3c",
     "2.0.21": "72f4ec97fbc7ec16c15ae68a75b0a257b2835160",
-    "2.0.20-lto": "d1b26cd17e51c5c705eea69b9545975e3705c058",
     "2.0.20": "e0c15cd14170f407a9eb27fcbad22931dc67feb7",
-    "2.0.19-lto": "4487f6c5107e7882ae2bad6d26c34ffdceb713f0",
+    "2.0.20-lto": "d1b26cd17e51c5c705eea69b9545975e3705c058",
     "2.0.19": "9b9ff2dabfb4a7fbacbc004c0bead12a60f9d05c",
+    "2.0.19-lto": "4487f6c5107e7882ae2bad6d26c34ffdceb713f0",
```

Comment:
I normally revert these re-ordering.. we should investigate why they happen.

Reply:
Should we just do the reordering in a different PR? does it actually matter?

Reply:
I just depends if the re-ordering is stable, or if it going to change ever time we run the script on different machine.  I never got around to figuring out if its just dict-ordering or something else.    I think recent python versions have stable dict order but I don't know if that is all this is.